recirq/qml_lfe/learn_dynamics_C.py

```diff
@@ -0,0 +1,167 @@
+import os
+import cirq
+import cirq_google
+import numpy as np
+import circuit_blocks
+
+from absl import app
```

Comment:
we need to add this as a dependency

Reply:
I added a `requirements.txt` file to the directory. CI doesn't seem to picking it up yet, am I missing an entry I need to make somewhere else for things to get picked up ?

Reply:
Ahhh changing to extra-requirements and adding an entry in setup.py fixed things.